Xamarin.Forms.Platform.MacOS/Renderers/ScrollViewRenderer.cs

```diff
@@ -19,10 +19,10 @@ public class ScrollViewRenderer : NSScrollView, IVisualElementRenderer
 
 		public ScrollViewRenderer() : base(RectangleF.Empty)
 		{
+            ContentView = new FlippedClipView();
 			DrawsBackground = false;
-			ContentView.PostsBoundsChangedNotifications = true;
-			NSNotificationCenter.DefaultCenter.AddObserver(this, new Selector(nameof(UpdateScrollPosition)),
-				BoundsChangedNotification, ContentView);
+            ContentView.PostsBoundsChangedNotifications = false;
+			NSNotificationCenter.DefaultCenter.AddObserver(this, new Selector(nameof(UpdateScrollPosition)), BoundsChangedNotification, ContentView);
```

Comment:
ok checked in.  I am beginning to think my merge tool is somehow adding spaces in.  I am going to reset all my settings and see if there is an option for tabs.  Can I ask what merge tool you guys use and are there specific settings?  I unfortunately am using 4 different computers with different VS instances from 2015, 2017, 2017 Preview, and VS macOS.  I need a different version at the current moment for each store deploy I make, for our in house code to xamarin's code.  Its getting difficult to track down which instance causes the merge issues as I now have merge tools for windows and a different one for macOS.  Its a mess!  I would love to have one singular approach or at least eliminate the problem child.